dev/prod_builders.json

```diff
@@ -845,6 +845,48 @@
          "repo": "flutter",
          "task_name": "win_web_tool_tests",
          "flaky": true
+      },
+      {
+        "name": "Windows_android complex_layout_win__compile",
+        "repo": "flutter",
+        "task_name": "complex_layout_win__compile",
+        "flaky": false
+      },
+      {
+        "name": "Windows_android basic_material_app_win__compile",
+        "repo": "flutter",
+        "task_name": "basic_material_app_win__compile",
+        "flaky": false
+      },
+      {
+        "name": "Windows_android flutter_gallery_win__compile",
+        "repo": "flutter",
+        "task_name": "flutter_gallery_win__compile",
+        "flaky": false
+      },
+      {
+        "name": "Windows_android windows_chrome_dev_mode",
```

Comment:
Did we enable these builds in flutter/infra for this branch?

Reply:
I do not know how to check that.

Reply:
Yep, the release branches copy the master builders, so this PR added these builders for all release branches: https://github.com/flutter/infra/pull/410/files